_haikus/2022_hacktoberfest_silhouette.md

```diff
@@ -0,0 +1,10 @@
+---
+layout: haiku
+title: 2022 Hackoberfest Haiku
+author: Silhouette
+---
+Silhouette fades in 
+
+Grabs keyboard to start PR
+
+Viva Hacktoberfest!
```

Comment:
```suggestion
Viva Hacktober!
```

Reply:
thank you for your comments - understood my mistake